src/js/core/widget/core/DatePicker.js

```diff
@@ -0,0 +1,1844 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-script-url */
+/* eslint-disable camelcase */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-redeclare */
+/* eslint-disable block-scoped-var */
+/* eslint-disable one-var */
+/* eslint-disable vars-on-top */
+/* eslint-disable indent */
+/* eslint-disable no-eq-null */
+/*global window, define, ns */
```

Comment:
Why did you disable it so many? Can't you fix it in code?

Reply:
We modified the lint to reduce the disabled count.
Unfortunately, for cases that are difficult to fix, you need to disable it.
I wrote about the necessary cases.